<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWidget</class>
 <widget class="QWidget" name="DepositWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bank Deposits</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="accountLabel">
         <property name="text">
          <string>Bank Account</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>bankComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="bankComboBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="cashLabel">
         <property name="text">
          <string>Cash deposits</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>cashDoubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="checksLabel">
         <property name="text">
          <string>Checks deposits</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>checksDoubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="visaLabel">
         <property name="text">
          <string>Credit cards deposits</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>visaDoubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="otherLabel">
         <property name="text">
          <string>Other deposits
(virements)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>otherDoubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="commentLabel">
         <property name="text">
          <string>Comment</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>commentEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="3">
        <widget class="QLineEdit" name="commentEdit"/>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="cashDoubleSpinBox">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="checksDoubleSpinBox"/>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="visaDoubleSpinBox"/>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="otherDoubleSpinBox"/>
       </item>
       <item row="4" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QDateEdit" name="beginDateEdit"/>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QDateEdit" name="endDateEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="beginDateLabel">
         <property name="text">
          <string>Begin date</string>
         </property>
         <property name="buddy">
          <cstring>beginDateEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="endDateLabel">
         <property name="text">
          <string>End date</string>
         </property>
         <property name="buddy">
          <cstring>endDateEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QComboBox" name="usersComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="usersLabel">
         <property name="text">
          <string>Users</string>
         </property>
         <property name="buddy">
          <cstring>usersComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="balanceDoubleSpinBox"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="balanceLabel">
         <property name="text">
          <string>Last balance</string>
         </property>
         <property name="buddy">
          <cstring>balanceDoubleSpinBox</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>108</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>&amp;Record</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="balanceButton">
         <property name="text">
          <string>&amp;Balance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>&amp;Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
